hadoop-ozone/dist/src/main/smoketest/s3/commonawslib.robot

```diff
@@ -23,7 +23,9 @@ ${ENDPOINT_URL}                http://s3g:9878
 ${OZONE_S3_HEADER_VERSION}     v4
 ${OZONE_S3_SET_CREDENTIALS}    true
 ${BUCKET}                      generated
+${BUCKET1}                     generated
 ${BUCKET_LAYOUT}               OBJECT_STORE
+${BUCKET_LAYOUT1}              FILE_SYSTEM_OPTIMIZED
```

Comment:
The naming of ${BUCKET_LAYOUT}  and ${BUCKET_LAYOUT1}  can also be changed to  ${BUCKET_LAYOUT_OBS} and ${BUCKET_LAYOUT_FSO} respectively.

Reply:
+1